File: 1 - Iniciante/C99/BEE1165.c
#include <stdio.h>

int main(void){
	int casos, num;
	int i, j, divisores;
	
	scanf("%d", &casos);
	
	for (i=1; i<=casos; i++){
		
		scanf("%d", &num);
		
		divisores = 0;
		for (j=1; j<=num; j++){
			if (num % j == 0)
				divisores++;
		}
		
		if (divisores > 2)
			printf("%d nao eh primo\n", num);
		else 
			printf("%d eh primo\n", num);
		
	} 
	
	return 0;
}

File: 1 - Iniciante/Python 3.11/BEE1165.py
cases = int(input())

for _ in range(cases):
    num = int(input())
    div = [i for i in range(1, num+1) if (num%i)==0]
    print(f"{num} nao eh primo") if (len(div)>2) else print(f"{num} eh primo")